tests/format1.f90

```
@@ -99,14 +99,16 @@ program format_04
     ! test for issue: https://github.com/lfortran/lfortran/issues/4040
     print "(2 (I3))", i, j
     print 25, i, j
-    25 FORMAT (F10.3)
```

this was the reason for failure of CI earlier, I added a wrong test for `(F10.3)` for integer values (i.e. `i, j`). We should raise an error anyway (that's what GFortran does as well) instead of passing without error (which we currently do).